Makefile

```diff
@@ -78,6 +78,9 @@ CTK_TAG ?= 91ee4968b7b97996f8c466a3ebbdce41168118e3
 # additional arguments to pass to 'docker build'
 BUILD_ARGS ?=
 BUILD_ARGS := $(BUILD_ARGS) --build-arg CTK_TAG=${CTK_TAG}
+BUILD_ARGS := $(BUILD_ARGS) --build-arg http_proxy=${http_proxy}
```

Comment:
@astronaut0131 The BUILD_ARGS make variable is defined to serve this purpose only. So, one can call:

`make docker-build BUILD_ARGS="--build-args http_proxy=someproxy --build-args no_proxy=skipproxy,list"`

Reply:
> Good idea, I think proxy setting can be mentioned in the doc, I've wasted some time on it actually.

Yes, I agree. Would be useful to document all the supported make variables.